runtime/src/main/java/org/corfudb/runtime/object/transactions/SnapshotTransactionalContext.java

```diff
@@ -85,4 +85,9 @@ public SnapshotTransactionalContext(Transaction transaction) {
     public void addTransaction(AbstractTransactionalContext tc) {
         throw new UnsupportedOperationException("Can't merge into a readonly txn (yet)");
     }
+
+    @Override
+    public void addPreCommitListener(TransactionalContext.PreCommitListener preCommitListener) {
```

Comment:
please add TransactionalContext to imports

Reply:
done. thanks